### DISCUSSION TITLE
explanation about project services

### DISCUSSION THREAD

**Discussion by redaabdellah21** [DependencyTrack/dependency-track#1638] [General]
hi, 
i started using DT lately and i noticed that projucts have services. yet, i can't find doc about what these services are and how i can define them. is there any way to explain them please? or redirect me to a doc.

thanks in advance.

**Comment by stevespringett** [comment]
Services are part of a BOMs inventory. The CycloneDX BOM format supports services in addition to software and hardware assets.

See https://cyclonedx.org/use-cases/#service-definition for an explanation and examples.

There's also a BOM containing only the services used by DT which you can import and experiment with.
https://github.com/DependencyTrack/dependency-track/blob/master/src/main/resources/services.bom.json

**Reply by msymons** [reply]
@stevespringett,  I tried importing the DT services BOM into DT and nothing visibly changed.  What is the correct usage?
* Use `cyclonedx-cli` tool and merge with DT BOM before import?
* Import  services BOM to existing project that has already had DT BOM imported? (this is what I tried)
* Import into a new project in DT?
* Something else?

**Reply by stevespringett** [reply]
There's a defect in DT - it assumes a BOM will have components and not just services. Fixing that now.

**Comment by redaabdellah21** [comment]
This will be very helpful. Thank you so much
